src/condor_scripts/condor_upgrade_check

```diff
@@ -718,6 +718,43 @@ def check_desktop_policy():
     return 0
 
 
+def check_pslot_default():
+    """Check for use of RANK in startd config due to incompatibility with
+    P-Slots on by default in V23.0.0"""
+    htc_version = htcondor.param["CONDOR_VERSION"].split(".")
```

Comment:
```suggestion
```
This is superseded by the next line.

Reply:
Good catch. don't know how I managed that